### PR TITLE
BSSwF: add nitin resources

### PR DESCRIPTION
**Pull request by gonsie**
add Nitin's links

### CODE REVIEW

**Review by bartlettroscoe — APPROVED** [review]
LGTM

File: Site/BSSwFellowshipProgram/People/SukhijaNi.md [hunk 1]
```diff
@@ -10,14 +10,19 @@
...
 
 Nitin Sukhija is an associate professor in the department of computer science and director of Center for Cybersecurity and Advanced Computing (C2AC) at Slippery Rock University of Pennsylvania. His areas of expertise are scientific computing focusing on performance modeling, robustness and resilience analysis, and cybersecurity.
+
+### Selected Resources
+<a href="https://bssw.io/blog_posts/secure-software-programming-practices-and-development" class="link-row">Secure Software Programming Practices and Development blog article</a>
+<a href="https://zenodo.org/communities/secure-scientific-software-development/" class="link-row">Best Practices and Tools for Secure Scientific Software Development</a>
+<a href="https://ideas-productivity.org/events/hpcbp-081-securesoftware" class="link-row">HPC Best Practices Webinar - Secure Software Programming Practices and Development</a>
```
Comment (bartlettroscoe) [inline Site/BSSwFellowshipProgram/People/SukhijaNi.md:28]: Actually, looking at the rendered page [here](https://github.com/gonsie/bssw.io/blob/bsswf22-updates/Site/BSSwFellowshipProgram/People/SukhijaNi.md), I think we want to provide bullets for these references.  Right now they just all run together.

**Review by bartlettroscoe — CHANGES_REQUESTED** [review]
I think we need to add bullets before the references.

File: Site/BSSwFellowshipProgram/People/SukhijaNi.md (hunk 1 above)
Comment (gonsie) [reply Site/BSSwFellowshipProgram/People/SukhijaNi.md:28]: Hi @bartlettroscoe, I think the CSS and HTML class "link-row" will do that for us. See:
- Full render: https://bssw.io/fellows/julia-stewart-lowndes
- Source: https://github.com/gonsie/bssw.io/blob/bsswf22-updates/Site/BSSwFellowshipProgram/People/LowndesJu.md

Comment (bartlettroscoe) [reply Site/BSSwFellowshipProgram/People/SukhijaNi.md:28]: @gonsie, I just rebuilt the preview site with this branch merged to the 'preview' branch and you are correct.  As shown at:

* https://preview.bssw.io/fellows/nitin-sukhija

and it shows:

image

But is is unfortunate that the direct Markdown rendering of these pages like in GitHub does not show this structure ☹️

**Review by bartlettroscoe — APPROVED** [review]
I built the preview site with this branch merged and it produced:

* https://preview.bssw.io/fellows/nitin-sukhija

and the article looks good.  We can go ahead and merge this.

**Pull request merged**